# exceptions.py

class FailedLogin(Exception):
    def __init__(self, username, password, response):
        self.username = username
        self.password = password
        self.resposne = respose

    @property
    def message(self):
        m = 'Failed to log (%s, %s) in. Response: %s' %
            (self.username, self.password, self.response)
        return m
        
